docs/installation/updating-with-npm.md

```diff
@@ -0,0 +1,39 @@
+# Updating GOV.UK Frontend with node package manager (NPM)
```

Comment:
I don't know why, but '_with_ node package manager' just doesn't sound quite right to me. Have you considered 'using' instead?

```suggestion
# Updating GOV.UK Frontend using node package manager (NPM)
```

Reply:
Good point - I'll tweak this.